Source/Utils/WarpXAlgorithmSelection.cpp

```diff
@@ -55,6 +55,12 @@ const std::map<std::string, int> electrostatic_solver_algo_to_int = {
     {"default", ElectrostaticSolverAlgo::None }
 };
 
+const std::map<std::string, int> poisson_solver_algo_to_int = {
+    {"multigrid", PoissonSolverAlgo::Multigrid},
+    {"fft-based", PoissonSolverAlgo::IntegratedGreenFunction},
```

Comment:
Just curious: Why do we call this FFT-based and not IntegratedGreenFunction/IGF?

Reply:
I think "Integrated Green Function" is not very common. 
On the other hand, "FFT-based" conveys the most important feature: that is a spectral electrostatic solver. 
If we used "IGF" I believe the users would have to look it up, while with FFT-based they would already have an idea of how the solver works.

Reply:
I agree with @aeriforme.
In addition, it fits better with the PICMI interface for the electrostatic solver:
https://picmi-standard.github.io/standard/field.html#electrostatic
Btw, actually, @aeriforme, could we use the exact same names as in PICMI ? (i.e. `Multigrid` and `FFT`)

Reply:
Sounds good to me, that would be consistent and intuitive.